pkgs/tools/admin/wander/default.nix

```diff
@@ -2,24 +2,17 @@
 
 buildGoModule rec {
   pname = "wander";
-  version = "0.8.0";
+  version = "0.8.2";
 
   src = fetchFromGitHub {
     owner = "robinovitch61";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-G/TrfnmEyomdUCN5nUS9v5iqeUzgZzMLUZnfroQLZuk=";
+    sha256 = "sha256-BcjK1GNj6URk6PmZIqG/t6vvy5ZXo3Z6wDqY1kbLSfw=";
   };
 
   vendorSha256 = "sha256-iTaZ5/0UrLJ3JE3FwQpvjKKrhqklG4n1WFTJhWfj/rI=";
 
-  patches = [
-    (fetchpatch {
```

Comment:
f
Please remove the unused fetchpatch from inputs.

Reply:
#205662